client/src/actions.js

```diff
@@ -0,0 +1,32 @@
+let socket
+let _initialized = false
+
+function initialized() {
+    return _initialized
```

Comment:
Maybe you should export only a top level function that takes everything necessary to become "initialized" and stores it in local variables. This top level function can then return the equivalent of Actions.